Calculator/CalculatorLogic/CalculatorLogic.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System.ComponentModel;
+
+namespace CalculatorLogic
+{
+    public class CalculatorLogic
+    {
+        double firstNumber;
+        double secondNumber;
+        public string CurrentOutput { get; set; }
```

Comment:
```suggestion
        public string CurrentOutput { get; private set; }
```
Зачем позволять редактировать свойство, если не предполагается, что её будут редактировать извне.